extensions/enrichedScatterplots.py

```diff
@@ -63,12 +64,30 @@ def main():
     x = [np.mean([float(negD[sn][pn]) for sn in negNames]) for pn in peptideNames]
     
```

Comment:
Before starting to process the input, please run a couple checks:

1. Make sure that the user didn't provide both the '-e' and '-f' options. If they did, please issue a warning saying that the '-f' option will be used and the '-e' option will be ignored. 
2. Make sure the user provided either the '-e' or '-f' option. If not, issue a warning stating that at least one needs to be provided.